src/modules/tools/zprobe/ZGridStrategy.cpp

```diff
@@ -327,24 +332,19 @@ bool ZGridStrategy::saveGrid() //std::string *args)
 
 }
 
-bool ZGridStrategy::loadGrid()
+bool ZGridStrategy::loadGrid(std::string args)
```

Comment:
inefficient.. should be const std::string& args, or better still const char* args)